ntoskrnl/se/tokencls.c

```diff
@@ -16,6 +16,13 @@
 
 /* INFORMATION CLASSES ********************************************************/
 
+/* FIXME: Delete this */
+typedef enum _TOKEN_ELEVATION_TYPE {
```

Comment:
Does this exist on Windows XP or 2003? Or is this Vista+?

Reply:
I found this code in our `winnt_old.h`.

Reply:
https://learn.microsoft.com/en-us/windows/win32/api/winnt/ne-winnt-token_elevation_type
![image](https://user-images.githubusercontent.com/2107452/234844545-90cc7dde-ebd5-4545-b692-9fc514a0e590.png)

Reply:
OK then why this whole ordeal of elevation for the AV is even needed, if this doesn't exist on Win2k3 ?

Reply:
> I couldn't download the target AV software.

See link in updated description of the JIRA report.

Reply:
![VirtualBox_ReactOS_27_04_2023_21_16_59](https://user-images.githubusercontent.com/2107452/234860114-c63b9b19-67cf-495c-94c4-ead5abdc0399.png)
It displayed this window for a while. And some minutes later, it vanished.

Reply:
What about Win2k3 ?